test/extended/operators/olm.go

```diff
@@ -70,4 +72,69 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			}
 		})
 	}
+
+	// OCP-22070 author: jiazha@redhat.com
+	g.It("[ocp-22070]support grpc sourcetype", func() {
+		buildPruningBaseDir := exutil.FixturePath("testdata", "olm")
+		catalogsourceConfig := filepath.Join(buildPruningBaseDir, "image-catalogsource.yaml")
+		subscriptionConfig := filepath.Join(buildPruningBaseDir, "image-sub.yaml")
+
+		createResourceByYAML(oc, catalogsourceConfig)
+		time.Sleep(30 * time.Second)
```

Comment:
Can we remove this sleep by polling or `watch`ing a resource instead?

Reply:
Yes, thanks, updated it.